setup.py

```diff
@@ -0,0 +1,23 @@
+import setuptools
```

Comment:
I'd recommend using a `pyproject.toml` file instead of a `setup.py`. It allows to store the configuration of various linters (pytest, black, isort, pylint, ....) and packaging information as well.

Reply:
@SauravMaheshkar would you be willing to make a PR into this branch to introduce `pyproject.toml`? I've never used it, don't know exactly how to get it working 😅

Reply:
Sure more than happy to do it 😄

Reply:
Added in [dd04183](https://github.com/soumik12345/mirnetv2/pull/13/commits/dd04183789b99a5e74480084beda2bd416fb030c)